commit 5032ccd72f502d48c18add97bed0687d333a01db
Author: Mart Raudsepp <leio@gentoo.org>
Date:   Sun Sep 27 20:48:07 2020 +0200

    build: always use internal synctex instead of automagically
    
    We don't have a separate synctex package and would have to pull in
    all of texlive-core; so instead of fixing the automagic use of it,
    just remove support for external synctex, as that's easier.

diff --git a/meson.build b/meson.build
index 66ebe2e8..a731b1b2 100644
--- a/meson.build
+++ b/meson.build
@@ -249,8 +249,7 @@ gsettings_desktop_schemas_dep = dependency('gsettings-desktop-schemas', required
 config_h.set('HAVE_DESKTOP_SCHEMAS', gsettings_desktop_schemas_dep.found())
 
 # *** libsynctex ***
-synctex_dep = dependency('synctex', version: '>= 1.19', required: false)
-external_synctex = synctex_dep.found()
+external_synctex = false
 
 # *** Mime types list ***
 mime_types_list = {
